test/e2e/apps/job.go

```diff
@@ -111,10 +108,7 @@ var _ = SIGDescribe("Job", func() {
 		framework.ExpectNoError(err, "failed to ensure job completion in namespace: %s", f.Namespace.Name)
 	})
 
-	// Requires the alpha level feature gate SuspendJob. This e2e test will not
-	// pass without the following flag being passed to kubetest:
-	//   --test_args="--feature-gates=SuspendJob=true"
```

Comment:
Did you add it to the infra repo? Don't forget to remove it.

Reply:
yep, once this PR is merged

Reply:
ref: https://github.com/kubernetes/test-infra/pull/22188